prep-css-syntax/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>CSS Syntax</title>
+  <style> h1{font-family: Arial; Helvetica; sans-serif}
+  p{font-size: 2rem} h3{color:firebrick} h2{font-style: italic}
+  </style>
+</head>
+<body>
+  <h1 style="font-family: arial">This h1 tag has been given Arial as a font-family.</h1>
+  <p style="font-size: 2rem">I am a p tag which has a font-size of 2rem!!!</p>
+  <h3 style="color: firebrick">This h3 tag has a font color of firebrick.</h3>
```

Comment:
To get this working you should be able to omit the `style` attributes from your elements.

Reply:
Oh okay! I see what you mean, Tim! 
Thank you! :)